src/main/java/org/teamresistance/frc/Gyro3D.java

```diff
@@ -0,0 +1,45 @@
+package org.teamresistance.frc;
+
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+import org.strongback.components.AngleSensor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Gyro3D implements AngleSensor {
+
+  private final AHRS gyro;
+
+  public Gyro3D(SPI.Port port) {
+    this.gyro = new AHRS(port);
+  }
+
+  @Override
+  /**
+   * Equivalent of getYaw()
+   */
+  public double getAngle() {
```

Comment:
`@Override` belongs after the JavaDoc:

```java
/**
 * Equivalent of getYaw()
 */
@Override
public double getAngle() {
```

PS: Thanks for having that comment.